auto_round/mllm/autoround_mllm.py

```diff
@@ -150,13 +150,15 @@ def __init__(
         self.to_quant_block_names = to_quant_block_names
         self.extra_data_dir = extra_data_dir
         self.quant_nontext_module = quant_nontext_module
+        self.processor = processor
+        if not hasattr(self.processor, "chat_template"):
```

Comment:
could move this to mllm example, Place before calling save quantize to prevent it from affecting tuning